src/data/local/github/tracks/track-2/steps/step-2.md

```diff
@@ -1,3 +1,15 @@
 # Step 2 - Create a simple Hello World
+#### Estimated time: 3 hours
 
-- [ ] Follow a tutorial of your choice to create a minimal "Hello World" project
+
+Follow a tutorial of your choice to create a minimal "Hello World" using the technology stack you were assigned.
+
+Examples of how getting started a new project with:
+- [React](https://github.com/Microsoft/TypeScript-React-Starter)
+- [Android](https://developer.android.com/training/basics/firstapp/creating-project)
+- [iOS](https://developer.apple.com/library/archive/referencelibrary/GettingStarted/DevelopiOSAppsSwift/BuildABasicUI.html#//apple_ref/doc/uid/TP40015214-CH5-SW1)
+
+
+**Note:** For now on, some steps will have an **estimated time** to help you organising your time better. 
+If you not finish the step in the estimated time **don't worry**! It's just an average time based on previous onboards.
```

Comment:
`If you do not finish`.

Que tal adicionar algo como: `Use this estimated time as a guide to evaluate when you should ask for help`?

Reply:
@tibawatanabe  na linha de baixo está escrito `By the way, remember that if you're struggling with any topic, feel free asking any Taqtiler ;)` Seria isso? 🤔

Reply:
Mais ou menos, a questão é o tempo que se fica tentando

Reply:
De qualquer forma @betopompolo, achei a sentença do Tiba melhor, principalmente por esclarecer o motivo dessas estimativas.